hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -397,6 +397,12 @@ class TrainerParams(param.Parameterized):
         param.Number(default=None,
                      doc="PyTorch Lightning trainer flag 'limit_test_batches': Limit the test dataset to the "
                          "given number of batches if integer, or proportion of test dataset if float.")
+    pl_limit_batches: Optional[IntOrBool] = \
+        param.Number(default=False,
+                     doc="PyTorch Lightning trainer flag 'fast_dev_run': Runs n if set to ``n`` (int) else 1 if set to"
```

Comment:
I would leave out the ability to set that to True/False - param.Number will prevent that anyway.